draft-holmberg-mmusic-t140-usage-data-channel.xml

```diff
@@ -318,24 +318,51 @@ Answer:
-          <t>PSTN text telephony</t>
-        </list>
+        A number of real-time text transports and protocols have been defined for
+        both packet switched and circuit switched networks. Many are based on the 
+        ITU-T T.140 protocol on application and presentation level <xref target="T140"/>. 
+        At the time of writing this document, some mechanisms are no longer used, as
+        the technologies they use have been obsoleted etc, while others are still in use.
       </t>
       <t>
-        In addition to simply moving text between two transport mechanisms, a gateway
-        might have to perform procedures related to events like inactivity of T.140 traffic,
-        RTP packets received out of order and loss of incoming RTP packets.
+        When performing interworking between T.140 data channels and another real-time text
+        transports and protocols with real-time text in another, a number of factors 
+        need to be considered.
+        At the time of writing this document, the most common IP-based real-time text transport
+        is the RTP based mechanism defined in <xref target="RFC4103"/>. While this document
+        does not define a complete interworking solution, this list below provides some guidance
+        and considerations to take into account when designing an gateway for interworking
+        between T140 data channels and RTP-based T.140 transport:
+        <list style="symbols">
+          <t>
+            For each T.140 data channel there is an RTP stream for real-time text <xref target="RFC4103"/> . 
+            Redundancy is by default declared and used on RTP stream. On the T.140 data channel
+            there is no redundancy, but the reliable property <xref target="I-D.ietf-mmusic-data-channel-sdpneg"/>
+            of T.140 the data channel is set.
+          </t>
+          <t>
+            During a normal text flow, T140blocks received from one network are forwarded towards the other network.
+            Keep-alive traffic is implicit on the T140 data channel. <xref target="RFC4103"/> does not define
+            keep-alive traffic for the RTP stream.
+          </t>
+          <t>
+            It is RECOMMENDED that the gateway uses the same transmission interval on both the T140 data channel and the RTP stream,
+            if possible. That will reduce the delay caused by buffering.
+          </t>
+          <t>
+            If the gateway detects or suspects loss of data on the RTP stream, the gateway gateway SHOULD insert the T.140 missing
+            text marker <xref target="T140ad1"/> in the data sent on the T.140 data channel. 
+          </t>
+          <t>
+            If the gateway detects that the T.140 data channel has failed, once the data channel has been reestablished
+            the gateway SHOULD insert the T.140 missing text marker <xref target="T140ad1"/> in the data sent on the RTP stream.
```

Comment:
This is fuzzy. I think directions need to be included in the sentences. And consideration if anything was lost.

Reply:
Doesn't "insert the T.140 missing text marker in the data sent on the RTP stream" indicate the direction? But, to make it more clear, I can add ", towards the RTP endpoint."